README.md

```diff
@@ -83,7 +89,27 @@ See [customize options] and [manage connection profiles] for more details.
 ```
 
 ## Change Log
-The current version is ```0.2.0```. See the [change log] for more detail.
+The current version is ```0.2.0```. See the [change log] for a detailed list of changes in each version.
+
+## Supported Operating Systems
+
+Currently this extension supports the following operatings systems:
+
+* Windows (64-bit only)
```

Comment:
I'm matching the c# extension's doc. We don't discriminate (and at least try to install & run). I'd like to wait until our first Issue of "this fails on RHEL 4" is raised).